.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,258 @@
+          - description: "Search"
+            test-id: "content-services"
+            folder: "search"
+            provider: "ECM"
+            auth: "BASIC"
+            check-cs-env: "true"
+          - description: "Process: Form"
+            test-id: "process-services"
+            folder: "process-services/form"
+            provider: "BPM"
+            auth: "OAUTH"
+            check-ps-env: "true"
+            check-external-cs-env: "true"
+          - description: "Process: Process"
+            test-id: "process-services"
+            folder: "process-services/process"
+            provider: "BPM"
+            auth: "OAUTH"
+            check-ps-env: "true"
+            check-external-cs-env: "true"
+          - description: "Process: Tasks"
+            test-id: "process-services"
+            folder: "process-services/tasks"
+            provider: "BPM"
+            auth: "OAUTH"
+            check-ps-env: "true"
+            check-external-cs-env: "true"
+          - description: "Process: Widget"
+            test-id: "process-services"
+            folder: "process-services/widgets"
+            provider: "BPM"
+            auth: "OAUTH"
+            check-ps-env: "true"
+            check-external-cs-env: "true"
+          - description: "Process Cloud: Form"
+            test-id: "process-services-cloud"
+            folder: "process-services-cloud/form-field"
+            provider: "ALL"
+            auth: "OAUTH"
+            apa-proxy: true
+            check-cs-env: "true"
+            check-ps-cloud-env: "true"
+          - description: "Process Cloud: People"
+            test-id: "process-services-cloud"
+            folder: "process-services-cloud/people"
+            provider: "ALL"
+            auth: "OAUTH"
+            apa-proxy: true
+            check-cs-env: "true"
+            check-ps-cloud-env: "true"
+          - description: "Process Cloud: Process"
+            test-id: "process-services-cloud"
+            folder: "process-services-cloud/process"
+            provider: "ALL"
+            auth: "OAUTH"
+            apa-proxy: true
+            check-cs-env: "true"
+            check-ps-cloud-env: "true"
+          - description: "Process Cloud: Start Task"
+            test-id: "process-services-cloud"
+            folder: "process-services-cloud/start-task"
+            provider: "ALL"
+            auth: "OAUTH"
+            apa-proxy: true
+            check-cs-env: "true"
+            check-ps-cloud-env: "true"
+          - description: "Process Cloud: Tasks List"
+            test-id: "process-services-cloud"
+            folder: "process-services-cloud/task-list"
+            provider: "ALL"
+            auth: "OAUTH"
+            apa-proxy: true
+            check-cs-env: "true"
+            check-ps-cloud-env: "true"
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0 # Fetch all history for all
```

Comment:
is it really necessary for e2e? will have an impact given how many jobs is launching

Reply:
nx affected requires to fetch everything before start and it's used by e2e tests